android/kotlin/AmpliApp/app/src/test/java/com/example/ampliapp/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.ampliapp
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
+    @Test
+    fun addition_isCorrect() {
```

Comment:
Please add basic unit tests (load, identify, group, track, event method)

Reply:
@justin-fiedler Do you mean we need new unit tests in addition to https://github.com/amplitude/ampli-examples/blob/AMP-44814-android-kotlin-example/android/kotlin/AmpliApp/app/src/androidTest/java/com/example/ampliapp/AmpliTest.kt?

Reply:
Sorry, current tests in androidTest are good.  Was confused why this file was empty.